binding/java/build.gradle

```diff
@@ -45,8 +45,9 @@ compileJava {
     options.compilerArgs.add("-Xlint:deprecation")
 }
 
+ext.seleniumVersion         = "3.0.0"
 project.archivesBaseName    = project.name
-project.version             = "1.3.0"
+project.version             = "$seleniumVersion-beta1"
```

Comment:
> shouldn't we use that one?

yes we should.  i changed it so the java client driver is the same version as the selenium dependency.